helms/odahu-flow-core/templates/operator/generated/odahuflow_v1alpha1_modelpackaging.yaml

```diff
@@ -39,6 +39,8 @@ spec:
               type: string
             image:
               type: string
+            outputConnection:
+              type: string
```

Comment:
The documentation is missing.

Reply:
https://github.com/odahu/odahu-docs/pull/33